hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -345,8 +392,10 @@ def from_typing_type(thing):
 
     try:
         # These aren't present in the typing module backport.
-        _global_type_lookup[typing.SupportsBytes] = st.binary()
-        _global_type_lookup[typing.SupportsRound] = st.complex_numbers()
+        _global_type_lookup[typing.SupportsBytes] = byteable_strategy()
```

Comment:
You could probably just inline the definition here, instead of having a single-use function.

Reply:
I needed to create a variable and I had some comments so i made a function. I agree it doesn't feel at home in the module. so i'm fine with removing and just duplicating the call to integers if it's preferred for coherence with surrounding code.

That said ....

At the moment, for the update statement alone  we will now be looking at about 42 lines including 6 lines of inline comments. I'd just say maybe putting some of the more complex strategies into helpers where they could be commented and all would not be such a bad thing?   I get that it's single use and so  a little silly but  but there's no harm really. 

Again I'm happy to inline it if that's preferred, since it doesn't fit current style.

Reply:
Doesn't really matter either way, so let's stick with the function :+1: